### PR TITLE
todotxt-net: Persist the right config file

### PR DESCRIPTION
**Pull request by dodorz**

### CODE REVIEW

File: bucket/todotxt-net.json [hunk 1]
```diff
@@ -13,9 +13,9 @@
             "todotxt.net"
         ]
     ],
-    "persist": "todotxt.exe.config",
+    "persist": "todotxt.settings",
```
Comment (Ash258) [inline bucket/todotxt-net.json]: This file do not exists in archive

File: bucket/todotxt-net.json [hunk 2]
```diff
@@ -6,6 +6,9 @@
     "url": "https://github.com/benrhughes/todotxt.net/releases/download/v3.3.1/todotxt-portable.zip",
     "hash": "1c0fe0d37d5519af0e701aebcd47ee17a39751386b18f8e216976c8f75bcf636",
     "extract_dir": "ProgramFiles",
+    "pre_install": [
```
Comment (linsui) [inline bucket/todotxt-net.json]:
```suggestion
    "pre_install": "if (!(Test-Path \"$persist_dir\\todotxt.settings\")) { New-Item \"$dir\\todotxt.settings\" -ItemType File | Out-Null }",
```

Comment (Ash258) [reply bucket/todotxt-net.json]: This would be fixed by Excavator

Comment (linsui) [reply bucket/todotxt-net.json]: Wow, thanks!

**Pull request merged**